lib/formatters/stringFormatter.js

```diff
@@ -87,15 +114,24 @@ function formatter(messages, source) {
 		(m) => m.column,
 	);
 
-	// Create a list of column widths, needed to calculate
-	// the size of the message column and if needed wrap it.
+	/**
+	 * Create a list of column widths, needed to calculate
+	 * the size of the message column and if needed wrap it.
+	 * @type {{[k: number]: number}}
+	 */
 	const columnWidths = { 0: 1, 1: 1, 2: 1, 3: 1, 4: 1 };
 
+	/**
+	 * @param {[string, string, string, string, string]} columns
+	 * @return {[string, string, string, string, string]}
+	 */
 	const calculateWidths = function(columns) {
 		_.forOwn(columns, (value, key) => {
+			/** @type {number} */
+			const k = /** @type {any} */ (key);
```

Comment:
Could explain this, please? `k` is type `number`, but at the same time it's type `any`, because of assigned variable.

Is it possible to add `@type {number}` to callback `key` argument instead of creating variable just to add types?

Reply:
fixed